\documentclass[11pt,letterpaper]{article}
\usepackage[utf8]{inputenc}
\usepackage[english]{babel}
\usepackage{cite}
\usepackage[margin=1in]{geometry}
\usepackage{paralist}

\title{Research Proposal}
\author{B. Arman Aksoy}
\date{}

\begin{document}

\maketitle

\section*{Making sense of cancer data: Implications on personalized therapy and basic biology}
\paragraph{Overview:} My research interests are interdiciplinary and lie at the intersection of Computational Biology and Cancer Genomics.
I am specifically interested in interpreting cancer data for discovering therapeutic opportunities and novel biological mechanisms that can further be experimentally investigated.
To this end, my research plans include
\begin{inparaenum}[(i)]
 \item developing computational methods to analyze cancer data for generating biological hypotheses (dry laboratory);
 \item using cell lines as models, testing these hypotheses in vitro with the help of basic biological techniques, such as genome editing and high-throughput profiling (wet laboratory)
\end{inparaenum}.

\paragraph{Motivation:} This line of research has recently become more feasiable as more and more data sets are becoming publicly available as a result of large-scale cancer genomics profiling and drug screening projects. 
The Cancer Genome Atlas (TCGA), Cancer Cell Line Encyclopedia (CCLE), Cancer Genome Project (CGP) and Cancer Therapeutics Response Portal (CTRP) are examples to near-complete data sets that are of interest in this context.
Initial results from these large-scale projects showed considerable amount of diversity in each cancer sample
and identified many novel therapuetic targets.
In addition to these major efforts, with the increased avability and decreased costs of cellular profiling, many cancer profiling projects are now producing data sets that are of comparable size to earlier major efforts.

These projects have provided unprecedented insights into cancer biology
and identified many frequent genomic alterations that seem to drive the process where a normal cell become a cancer one.
In the light of these results, new clinical trials are being established, recruiting patients based on their genomic profiles for a common therapeutic intervention.
A majority of the events that are observed in patients, however, remain unexplained.
This is partly due to lack of immediate therapeutic intervention for many alterations
and partly due to relatively low frequency of these events--hence lack of interest and experimental follow-up.

\paragraph{Past work: }I have been part of teams that develop computational utilities that help other researchers better 
investigate cancer genomics data (cBioPortal \cite{gao2014cbioportal, cerami2012cbio}),
integrate biological pathway data (Pathway Commons and BioPAX \cite{demir2013using,babur2014integrating})
and query available therapeutic targeted drugs (PiHelper \cite{aksoy2013pihelper}).
I have also helped other researchers in our group with integrating these data sources into their analyses \cite{ciriello2013emerging,korkut2014perturbation,babur2014systematic}.

Finally, by making use of these knowledge-bases, 
I showed that 
\begin{inparaenum}[(i)]
 \item random passenger genomic events can create patient-specific therapeutic vulnerabilities that can be exploited by targeted-drugs \cite{aksoy2014prediction};
 \item comprehensive analysis of cancer genomics data sets can reveal interesting biological insights about infrequent events \cite{aksoy2014cancer}
\end{inparaenum}.

\paragraph{Future work:} 
In the short term, I would like to experimentally validate computational predictions on therapeutic vulnerabilities and cancer biology in wet laboratory
and therefore establish an experimental workflow that can efficiently screen cell line models for these types of vulnerabilities (Aim 1-2).
In the long term, I would like to work on developing cancer therapy strategies that are tailored to a patient's genomic profile to increase the efficiancy of a therapy (Aim 3).

\subsection*{Research goals}
\paragraph{Aim 1: Identification and testing of individualized therapeutic vulnerabilities in cancer}
A majority of alterations observed in cancer samples occur at low-frequency across patient cohorts
and do not confer cells any proliferative advantage--therefore are called passenger alterations.
I previously showed that a considerable amount of passenger alterations create therapeutic vulnerabilities that can be exploited with the use of a targeted drug, creating a unique personalized medicine opportunity \cite{aksoy2014prediction}.
By making use of data from cancer cell lines,
I showed that a majority of the vulnerabilities we identified in cancer patients also exist in cell lines
and allow testing these predictions in cell line models, which was something I was not able to pursue in a computationally oriented lab.

As the next step, I will extend the computational approach to analyze more data sets (both tumor and cell line data) and also to cover the whole cellular pathways for identifying vulnerabilities of therapeutic interest.
I then will test the most common vulnerabilities that are seen in patients by introducing clinically relevant drugs to the matching cell line models that also have any of these common vulnerabilities.
In paralell, I will also utilize the recent genome editing tools (CRISPR/Cas9 in particular)
to screen for pairs of genes that are predicted to be synthetic lethal for the cell.
The computational prediction part of this work will significantly reduce the number of genes that are of interest,
and therefore will make it more feasible.


\paragraph{Aim 2: Exploring cancer-specific genomic alterations that reveal biological facts}
I did this \cite{aksoy2014cancer}.
Mutations in the RNase IIIb domain of DICER1 are known to disrupt processing of 5p-strand pre-miRNAs and these mutations have previously been associated with cancer. 
Using data from the Cancer Genome Atlas project, we show that these mutations are recurrent across four cancer types and that a previously uncharacterized recurrent mutation in the adjacent RNase IIIa domain also disrupts 5p-strand miRNA processing. 
Analysis of the downstream effects of the resulting imbalance 5p/3p shows a statistically significant effect on the expression of mRNAs targeted by major conserved miRNA families. 
In summary, these mutations in DICER1 lead to an imbalance in miRNA strands, which has an effect on mRNA transcript levels that appear to contribute to the oncogenesis.

\paragraph{Aim 3: Exploiting evolutionary dynamics of cancer cells for a more efficient targeted therapy}
I'm gonna do that.

\clearpage

\footnotesize{
\bibliography{proposal}{}
\bibliographystyle{plain}}
\end{document}
