\documentclass[12pt,letterpaper]{article}
\usepackage[utf8]{inputenc}
\usepackage[english]{babel}
\usepackage{cite}
\usepackage[margin=1in]{geometry}
\usepackage{paralist}

\title{Research Proposal}
\author{B. Arman Aksoy}
\date{}

\begin{document}

\maketitle

\section*{Making sense of cancer data: Implications on personalized therapy and basic biology}
My research interests are interdiciplinary and lie at the intersection of Computational Biology and Cancer Genomics.
I am specifically interested in interpreting cancer data for discovering therapeutic opportunities and novel biological mechanisms.
To this end, my research plans include
\begin{inparaenum}[(i)]
 \item developing computational methods to analyze cancer data for generating biological hypotheses;
 \item using cell lines as models, testing these hypotheses in vitro with the help of basic biological techniques, such as genome editing and high-throughput profiling
\end{inparaenum}.

This line of research has recently become more feasiable as more and more data sets are becoming publicly available as a result of large-scale cancer genomics profiling and drug screening projects. 
The Cancer Genome Atlas (TCGA), Cancer Cell Line Encyclopedia (CCLE), Cancer Genome Project (CGP) and Cancer Therapeutics Response Portal (CTRP) are examples to near-complete data sets that are of interest in this context.
Initial results from these large-scale projects showed considerable amount of diversity in each cancer sample
and identified many novel therapuetic targets.
In addition to these major efforts, with the increased avability and decreased costs of cellular profiling, many cancer profiling projects are now producing data sets that are of comparable size to earlier major efforts.

These projects have provided unprecedented insights into cancer biology
and identified many frequent genomic alterations that seem to drive the process where a normal cell become a cancer one.
In the light of these results, new clinical trials are being established, recruiting patients based on their genomic profiles for a common therapeutic intervention.
A majority of the events that are observed in patients, however, remain unexplained.
This is partly due to lack of immediate therapeutic intervention for many alterations
and partly due to relatively low frequency of these events--hence lack of interest and experimental follow-up.

I have been part of teams that develop computational utilities that help other researchers better 
investigate cancer genomics data (cBioPortal \cite{gao2014cbioportal, cerami2012cbio}),
integrate biological pathway data (Pathway Commons and BioPAX \cite{demir2013using,babur2014integrating})
and query available therapeutic targeted drugs (PiHelper \cite{aksoy2013pihelper}).
I have also helped other researchers in our group with integrating these data sources into their analyses \cite{ciriello2013emerging,korkut2014perturbation,babur2014systematic}.

Finally, by making use of these knowledge-bases, 
I showed that 
\begin{inparaenum}[(i)]
 \item random passenger genomic events can create patient-specific therapeutic vulnerabilities that can be exploited by targeted-drugs \cite{aksoy2014prediction};
 \item comprehensive analysis of cancer genomics data sets can reveal interesting biological insights about infrequent events \cite{aksoy2014cancer}.
\end{inparaenum}


\subsection*{Aim 1: Identification and testing of therapeutic individualized vulnerabilities in cancer}
\paragraph{Previous work:} 
I did this \cite{aksoy2014prediction}.
Somatic homozygous deletions of chromosomal regions in cancer, while not necessarily oncogenic, may lead to therapeutic vulnerabilities specific to cancer cells compared with normal cells. 
A recently reported example is the loss of one of the two isoenzymes in glioblastoma cancer cells such that the use of a specific inhibitor selectively inhibited growth of the cancer cells, which had become fully dependent on the second isoenzyme. 
We have now made use of the unprecedented conjunction of large-scale cancer genomics profiling of tumor samples in The Cancer Genome Atlas (TCGA) and of tumor-derived cell lines in the Cancer Cell Line Encyclopedia, as well as the availability of integrated pathway information systems, such as Pathway Commons, to systematically search for a comprehensive set of such epistatic vulnerabilities. 
Based on homozygous deletions affecting metabolic enzymes in 16 TCGA cancer studies and 972 cancer cell lines, we identified 4104 candidate metabolic vulnerabilities present in 1019 tumor samples and 482 cell lines. 
Up to 44\% of these vulnerabilities can be targeted with at least one Food and Drug Administration-approved drug. 
We suggest focused experiments to test these vulnerabilities and clinical trials based on personalized genomic profiles of those that pass preclinical filters. 
We conclude that genomic profiling will in the future provide a promising basis for network pharmacology of epistatic vulnerabilities as a promising therapeutic strategy.

\paragraph{Proposed research:}
I'm gonna do that.

\subsection*{Aim 2: Exploring cancer-specific genomic alterations that reveal biological facts}
\paragraph{Previous work:}
I did this \cite{aksoy2014cancer}.
Mutations in the RNase IIIb domain of DICER1 are known to disrupt processing of 5p-strand pre-miRNAs and these mutations have previously been associated with cancer. 
Using data from the Cancer Genome Atlas project, we show that these mutations are recurrent across four cancer types and that a previously uncharacterized recurrent mutation in the adjacent RNase IIIa domain also disrupts 5p-strand miRNA processing. 
Analysis of the downstream effects of the resulting imbalance 5p/3p shows a statistically significant effect on the expression of mRNAs targeted by major conserved miRNA families. 
In summary, these mutations in DICER1 lead to an imbalance in miRNA strands, which has an effect on mRNA transcript levels that appear to contribute to the oncogenesis.

\paragraph{Proposed research:}
I'm gonna do that.

\subsection*{Aim 3: Exploiting evolutionary dynamics of cancer cells for driving them to more vulnerable states}
\paragraph{Proposed work:}
I'm gonna do that.

\clearpage

\footnotesize{
\bibliography{proposal}{}
\bibliographystyle{plain}}
\end{document}
